src/client/locales/en-US.json

```diff
@@ -287,5 +287,12 @@
   "login_modal_footer_text": "Don't have an account? Signup with {link}",
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
-  "login_with_steemconnect": "Login with SteemConnect"
-}
\ No newline at end of file
+  "login_with_steemconnect": "Login with SteemConnect",
```

Comment:
I don't think we should add translations there.

Reply:
to en-US?

Reply:
Yeah, Crowdin uses `default.json` only.

Reply:
oh so we are gonna rely on crowdin for en-US translations?